src/progress/ux-progress.ts

```diff
@@ -0,0 +1,35 @@
+import {customElement, bindable, ViewResources, View, processAttributes} from 'aurelia-templating';
+import {inject} from 'aurelia-dependency-injection';
+import {StyleEngine} from '../styles/style-engine';
+import {Themable} from '../styles/themable';
+import {processDesignAttributes} from '../designs/design-attributes';
+
+@inject(ViewResources, StyleEngine)
+@customElement('ux-progress')
+@processAttributes(processDesignAttributes)
+export class UxProgress implements Themable {
+  @bindable public type = null;
+  @bindable public theme = null;
+  @bindable public barWidth = {

```

Comment:
I would very much like to see the width of the bar bound to `value` which will align with the standard `<progress>` HTML5 tag. The `progress` tag has two attributes: `value` which represents the current progress, and `max` which indicates the value that represents full.

`max` could have a default value of 100 so that it is not required. Then we can just calculate the percentage based on `value` and `max`. You could use a Computed Property to keep the HTML how it is.

Example
``` javascript
@computedFrom('this.max', 'this.value')
public get barWidth(){
    return `width: ${(this.value / this.max) * 100}%`;
}
```